### ISSUE TITLE
The Window Titrle Should Reflect Where in The App The User Is

### ISSUE THREAD

**Issue by kellylford** [kellylford/Scores#31]
Screen readers commonlhy use the window title to know where in an application you are. The window title here simply says sports scores. It should be updated to reflect where you are. For example, if the user selects mlb off the home screen, the title should say MLB - Sports Scores

This pattern should continue with the most unique item first. For example if the user selects standings after picking Sports Scores, it should say MLB, Standings - Sports Scores and so forth

Please test this comprehensively and show evidence of what testing was done as a part of code development.
